bundles/org.openhab.binding.androidtv/README.md

```diff
@@ -123,6 +123,18 @@ You may also send it a command of the app package name (e.g. com.google.android.
 KEYCODE values are listed at the bottom of this README.
 NOTE: Not all KEYCODES work on all devices.  Keycodes above 255 have not been tested.
 
+## Command Line Access
+
+All String type channels may receive commands from inside the karaf cli, even if there are no items configured.
+
+This can be particularly useful for the Pin Code Process as well as for testing.
+
+Syntax:
+openhab> openhab:androidtv <thingUID> <channel> <command>
+
+Example usage:
+openhab> openhab:androidtv androidtv:googletv:theater keypress KEY_POWER
+
 ## Pin Code Process
 
 For the AndroidTV to be successfully accessed an on-screen PIN authentication is required on the first connection.  
```

Comment:
Since this feature was requested to provide console commands for the PIN process, you should include examples of using console commands to request and submit the PIN.

Reply:
Good point.  Just pushed an update.